src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -119,7 +119,7 @@ body {
     ```css title="src/styles/global.css" ins={4}
     @import 'tailwindcss';
 
-    @theme {
+    @theme inline {
         --font-sans: var(--font-roboto);
     }
     ```
```

Comment:
If we do want to recommend `inline` for this reason or another, we should probably highlight it in this example.

Reply:
Perhaps a comment like

To prevent duplicate variable declarations, use inline (link to tw docs)

I'm not even sure if that's the right explanation though